packages/fast-usdc/src/exos/advancer.js

```diff
@@ -112,8 +106,9 @@ export const prepareAdvancerKit = (
         async handleTransactionEvent(evidence) {
           await null;
           try {
-            // TODO poolAccount might be a vow we need to unwrap
-            const { assetManagerFacet, poolAccount } = this.state;
+            const { assetManagerFacet } = this.state;
+            // TODO is there a better way than this?
+            const poolAccount = await when(this.state.poolAccount);
```

Comment:
make the caller pass an `OrchestrationAccount`, not a `Vow<OrchestrationAccount>`